docs/sphinx/developers/java-library.txt

```diff
@@ -58,14 +58,37 @@ for a complete and up-to-date list of all optional libraries, which can
 all be found in our :sourcedir:`Git repository <jar>`.
 
 Examples of usage
------------------
+=================
```

Comment:
You've now made this the same level as the page title - look at https://github.com/openmicroscopy/ome-documentation#title-headings for how the sphinx hierarchy works